tests/manage/mcg/test_object_expiration.py

```diff
@@ -0,0 +1,67 @@
+import logging
+import uuid
+from time import sleep
+
+import pytest
+
+from ocs_ci.framework.pytest_customization.marks import tier2, bugzilla
+from ocs_ci.framework.testlib import MCGTest
+from ocs_ci.framework.testlib import skipif_ocs_version
+from ocs_ci.ocs.bucket_utils import s3_put_object, s3_get_object
+
+logger = logging.getLogger(__name__)
+
+
+class TestObjectExpiration(MCGTest):
+    """
+    Tests suite for object expiration
+
+    """
+
+    @skipif_ocs_version("<4.10")
+    @bugzilla("2034661")
+    @pytest.mark.polarion_id("OCS-3929")
+    @tier2
```

Comment:
@parikshithb not sure if this really comes under tier2. Tier2 tests involves testing core features with some complexity involved in the steps and this test doesnt look to fall into that category
 Should we add it to tier1?

Reply:
ack